maven/lib/dependabot/maven/file_parser/repositories_finder.rb

```diff
@@ -23,7 +23,7 @@ class RepositoriesFinder
 
         # The Central Repository is included in the Super POM, which is
         # always inherited from.
-        CENTRAL_REPO_URL = "https://repo.maven.apache.org/maven2"
+        CENTRAL_REPO_URL = ENV['SETTINGS_CENTRAL_REPO_URL'] || "https://repo.maven.apache.org/maven2"
```

Comment:
```suggestion
        CENTRAL_REPO_URL = ENV['SETTINGS_MAVEN_CENTRAL_REPO_URL'] || "https://repo.maven.apache.org/maven2"
```

It might be worth including the package manager in the ENVVAR just to avoid any accidental collisions if core is being used in multi-ecosystem mode.